OG_Format.adoc

```diff
@@ -82,6 +82,7 @@ This document defines the common OceanGliders data format, hereafter OG1.0.
 * Format follows the ACDD 1.3footnote:[https://wiki.esipfed.org/Attribute_Convention_for_Data_Discovery_1-3] convention where possible.
 * Vocabulary collections will be hosted in different places (NERC Vocabulary Server - NVS, OceanOPS, ICES, etc). The OceanGliders data management team will manage (additions, updates, etc.) the collections. Treatment of vocabularies is detailed in the https://oceangliderscommunity.github.io/OG-format-user-manual/vocabularyCollection/tableOfControlledVocab.html[vocabularies document].
 * OG1.0 defines some variables and definitions, as described in this document. Other types of measurements (e.g. intermediate parameters, technical measurements, other variables) not framed by OG1.0 can be included in OG1.0 data files. No control will be applied to those measurements.
+* OG1.0 follows the IODE quality control, IODE_QCfootnote:[https://vocab.nerc.ac.uk/collection/L27/current/IODE_QC/], flagging schemafootnote:[https://repository.oceanbestpractices.org/handle/11329/413] 
```

Comment:
Perhaps a link to the DOI for the IODE flagging document would be better? http://dx.doi.org/10.25607/OBP-6

Reply:
pushed a new commit with this URL